var/spack/repos/builtin/packages/ultraliser/package.py

```diff
@@ -9,14 +9,11 @@
 class Ultraliser(CMakePackage):
     """Mesh and volume reconstruction of neuroscientific models
     """
-    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/viz/Ultraliser"
-    git = "ssh://bbpcode.epfl.ch/viz/Ultraliser"
+    homepage = "https://github.com/BlueBrain/Ultraliser"
+    git = "https://github.com/BlueBrain/Ultraliser"
```

Comment:
```suggestion
    git = "git@github.com:BlueBrain/Ultraliser.git"
```
it's still private, so https access does not work with `bbprelman`.

Reply:
Thanks for the suggestion @matz-e